app/src/main/java/explorer/city/com/cityexplorer/Model/AlternateName.kt

```diff
@@ -0,0 +1,5 @@
+package explorer.city.com.cityexplorer.Model
+
+data class AlternateName(
+        val name: String
```

Comment:
This should be on a single line. See here: https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting